omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/visit/ParserEncounterIntoSimpleObjects.java

```diff
@@ -131,6 +135,14 @@ private SimpleObject parseObs(Obs obs, Locale locale) {
         else {
             SimpleObject simpleObject = SimpleObject.create("obsId", obs.getObsId());
 
+            if (obs.getConcept().isNumeric()) {
+                Boolean integerNumber = false;
```

Comment:
Why not `allowDecimals`? So you stick to the concept numeric naming.

Reply:
But then I would always need to check the false case... That is the only reason...

Reply:
Changed to `isIntegerValue`